sentry-spring/src/main/java/io/sentry/spring/reactive/SentryReactiveErrorAttributes.java

```diff
@@ -0,0 +1,15 @@
+package io.sentry.spring.reactive;
+
+import com.jakewharton.nopen.annotation.Open;
+import org.springframework.boot.web.reactive.error.DefaultErrorAttributes;
+import org.springframework.web.server.ServerWebExchange;
+
+@Open
+public class SentryReactiveErrorAttributes extends DefaultErrorAttributes {
+
+  @Override
+  public void storeErrorInformation(Throwable error, ServerWebExchange exchange) {
+    SentryReactiveWebHelper.withRequestHub(exchange, iHub -> iHub.captureException(error));
```

Comment:
nit:

the `I` in it is just a convention to indicate it's an interface, but I suggest we just call it `hub` here:

```suggestion
    SentryReactiveWebHelper.withRequestHub(exchange, hub -> hub.captureException(error));
```